samples/csharp/roslyn-sdk/SemanticQuickStart/Program.cs

```diff
@@ -64,7 +58,7 @@ static void Main(string[] args)
             var systemSymbol = (INamespaceSymbol)nameInfo.Symbol;
             foreach (INamespaceSymbol ns in systemSymbol.GetNamespaceMembers())
             {
-                Console.WriteLine(ns.Name);
+                Console.WriteLine(ns);
```

Comment:
I'm curious why you made this change. I liked showing the distinction between `ns.ToString()` which returns the flly qualified name, and `ns.Name` which returns the scoped name. It's not a strong preference, but I'm curious.

Reply:
I think that knowing how to get the full namespace name is more useful and less discoverable than getting just the scoped name.

Also, I think it makes the output clearer: when I see the list `System.Collections`, `System.Configuration`, `System.Deployment`, I can clearly see those are namespace. When I saw `Collections`, `Configuration`, `Deployment`, it took me a second to figure out what I was looking at.

For those reasons, I prefer it this way, but it's not a very strong preference either.